### DISCUSSION TITLE
New install

### DISCUSSION THREAD

**Discussion by Julien4135** [robske110/IDDataLogger#39] [Q&A]
Following my new installation. Now I now have warnings in red :(

- > pi@raspberrypi:~/IDDataLogger $ ./start.sh
- > Loading Logger...
- > 22:40:39.423075 [INFO] Starting ID DataLogger Version v0.0.6-InDev (01c58d21e5b684fdbfe23661944cf09011904f3f)...
- > Loading Main...
- > 22:40:39.479487 [INFO] Reading config...
- > 22:40:39.479679 [INFO] Connecting to db...
- > Loading DatabaseConnection...
- > Loading ErrorUtils...
- > 22:40:39.493225 [CRITICAL] RuntimeException: "Failed to connect to db (check db connection params): SQLSTATE[08006] [7] FATAL: password authentication failed for user "iddatalogger" [7]" (EXCEPTION) in "/home/pi/IDDataLogger/src/vwid/db/DatabaseConnection.php" at line 77
- > 22:40:39.493446 [EMERGENCY] CRASHED WHILE STARTING; TRYING TO SHUTDOWN SAFELY
- > 22:40:39.493551 [INFO] Shutting down...
- > 22:40:39.493642 [CRITICAL] Forcibly shutting down while starting!
- > pi@raspberrypi:~/IDDataLogger $

**Comment by robske110** [comment]
Did any error pop up while you executed the [install.sh](https://github.com/robske110/IDDataLogger/blob/master/docs/install.sh) script?

What is the output of the command `service status postgresql`? If that looks fine, what does
```
sudo su postgres
psql
SELECT rolname FROM pg_roles;
```
show?
